### ISSUE TITLE
Bug: Unable to Save as Copy in any format

### ISSUE THREAD

**Issue by blewhew** [inkstitch/inkstitch#2739]
I've created an SVG from scratch. (And brand new to Inkscape for this project, so please bear with me if I've done something obviously wrong.) 

I was able to export an early version of my .svg as an .xxx (Singer embroidery) file, when it was just text, but now that I have the finished image, Inkscape won't 'Save as Copy' it in any format, embroidery or even as PNG. It's throwing this bug error message:

> Ink/Stitch Version: v3.0.1 (windows)

> Traceback (most recent call last):
>   File "shapely\geometry\multipolygon.py", line 189, in geos_multipolygon_from_polygons
> TypeError: 'Polygon' object is not subscriptable

> During handling of the above exception, another exception occurred:

> Traceback (most recent call last):
>   File "inkstitch.py", line 84, in <module>
>   File "inkex\base.py", line 250, in run
>   File "lib\extensions\output.py", line 56, in effect
>   File "lib\extensions\base.py", line 133, in elements_to_stitch_groups
>   File "lib\elements\element.py", line 556, in embroider
>   File "lib\elements\element.py", line 616, in validate
>   File "lib\elements\fill_stitch.py", line 608, in validation_errors
>   File "lib\elements\fill_stitch.py", line 542, in shape
>   File "lib\elements\fill_stitch.py", line 592, in _get_clipped_path
>   File "shapely\geometry\multipolygon.py", line 60, in __init__
>   File "shapely\geometry\multipolygon.py", line 191, in geos_multipolygon_from_polygons
> AttributeError: 'list' object has no attribute '_ndim'

![Walls of Jerusalem 1](https://github.com/inkstitch/inkstitch/assets/161090278/1b0ac789-ec9e-4202-b0a5-01555263d5ee)

**Comment by blewhew** [comment]
I tried it again on Mac, but same overall error.

**Comment by blewhew** [comment]
Thanks. I've read through that guide but I don't see what I would've done wrong/differently to get that error. Could you please offer any insight into what would be causing the issue; where I can re-do or fix it for it to work? 

I tried disabling each layer in turn and saving as an embroidery file, but I couldn't isolate any specific layer as an issue.

**Comment by kaalleen** [comment]
I think the problem is in the font. Fonts are a difficult issue in embroidery. Depending on size they mostly work best as satins. You could have a look at the font library (https://inkstitch.org/fonts/font-library/) to see if there is any existing font that may fit to your project (if you are ok with using a slightly different font and want to avoid a lot of work). Maybe Ink/Stitch small or Geneva simple might be a fit. Use only the scaling of the lettering tool. You can then use Lettering along Path (https://inkstitch.org/docs/lettering/#lettering-along-path) to bring it into shape.

**Comment by kaalleen** [comment]
Also you do not have to cut out every overlapping bit, but you want to reduce the size of some of the shapes to avoid huge overlapping areas.

**Comment by blewhew** [comment]
Thanks, @kaalleen. I was able to get it to work in the end! :pray:

I cleaned up the layers by deleting any points hanging outside the canvas, and cleaned up the overlapping shapes which I think made it more attractive, anyway. It was still bugging out out, though.

After more trial and error, removing the bottom line of text resolved the bug. So, I've got it working by removing and re-creating that line.
![Walls of Jerusalem 6](https://github.com/inkstitch/inkstitch/assets/161090278/0133ebb1-e5db-49b1-b6f6-789f68af5a2e)

**Comment by blewhew** [comment]
Thanks, @kaalleen. I was able to get it to work in the end! :pray:

I cleaned up the layers by deleting any points hanging outside the canvas, and cleaned up the overlapping shapes which I think made it more attractive, anyway. It was still bugging out, though.

After more trial and error, removing the bottom line of text resolved the bug. So, I've got it working by removing and re-creating that line.
![Walls of Jerusalem 6](https://github.com/inkstitch/inkstitch/assets/161090278/906e1b36-3d25-4313-9fe7-0d92bc6f09ef)

**Comment by blewhew** [comment]
Note to anyone running into something like this in future: Open the embroidery file in Inkscape to confirm it's converted correctly, before you try printing it.

I'm glad I did, because the text was converted wrongly. I had to redo BOTH lines of text for it to actually export correctly as a singer file. 

Here's the finalised .svg that actually worked 100% when exported as an .xxx file. :) 

![Walls of Jerusalem 7](https://github.com/inkstitch/inkstitch/assets/161090278/98130832-8c7c-43c0-9463-8c2ee0704125)
